examples/compute_polar_coordinates.py

```diff
@@ -0,0 +1,403 @@
+
+# add mean
+ax.axvline(
+    x=rho_data.mean().values,
+    c="b",
+    linestyle="--",
+)
+
+
+# add median
+ax.axvline(
+    x=rho_data.median().values,
+    c="r",
+    linestyle="-",
+)
+
+# add legend
+ax.legend(
+    [
+        f"mean = {np.nanmean(rho_data):.2f} pixels",
+        f"median = {np.nanmedian(rho_data):.2f} pixels",
+    ],
+    loc="best",
+)
+ax.set_ylabel("count")
+ax.set_xlabel("rho (pixels)")
+fig.show()
+
+# %%
+# We can see that there is some spread in the value of ``rho`` in this
+# dataset. This may be due to noise in the detection of the head keypoints,
+# or due to the mouse tipping its snout upwards during the recording.
+
+# %%
+# Histogram of ``phi`` values
+# -------------------------------------
+# We can also explore which ``phi`` values are most common in the dataset with
+# a circular histogram.
+
+# sphinx_gallery_thumbnail_number = 5
+
+# compute number of bins
+bin_width_deg = 5  # width of the bins in degrees
+n_bins = int(360 / bin_width_deg)
+
+# initialise figure with polar projection
+fig = plt.figure()
+ax = fig.add_subplot(projection="polar")
+
+# plot histogram using xarray's built-in histogram function
+head_vector_polar.sel(individuals=mouse_name, space_pol="phi").plot.hist(
+    bins=np.linspace(-np.pi, np.pi, n_bins + 1), ax=ax
+)
+
+# axes settings
+ax.set_title("phi histogram")
+ax.set_theta_direction(-1)  # set direction clockwise
+ax.set_theta_offset(0)  # set zero at the right
+ax.set_xlabel("")  # remove default x-label from xarray's plot.hist()
+
+# set xticks to match the phi values in degrees
+n_xtick_edges = 9
+ax.set_xticks(np.linspace(0, 2 * np.pi, n_xtick_edges)[:-1])
+xticks_in_deg = (
+    list(range(0, 180 + 45, 45)) + list(range(0, -180, -45))[-1:0:-1]
+)
+ax.set_xticklabels([str(t) + "\N{DEGREE SIGN}" for t in xticks_in_deg])
+
+fig.show()
+
+# %%
+# The ``phi`` circular histogram shows that the head vector appears at a
+# variety of orientations in this dataset.
+
+# %%
+# Polar plot of the head vector within a time window
+# ---------------------------------------------------
+# We can also use a polar plot to represent the head vector in time. This way
+# we can visualise the head vector in a coordinate system that translates with
+# the mouse but is always  parallel to the pixel coordinate system.
+# Again, this will be easier to visualise if we focus on a
+# small time window.
+
+# select phi values within a time window
+phi = head_vector_polar.sel(
+    individuals=mouse_name,
+    space_pol="phi",
+    time=time_window,
+).values
+
+# plot tip of the head vector within that window, and color based on time
+fig = plt.figure()
+ax = fig.add_subplot(projection="polar")
+sc = ax.scatter(
+    phi,
+    np.ones_like(phi),  # assign a constant value rho=1 for visualization
+    c=time_window,
+    cmap="viridis",
+    s=50,
+)
+
+# axes settings
+ax.set_theta_direction(-1)  # set direction counterclockwise
```

Comment:
the comment here says "counterclockwise", but in the previous plot the same line is designated as "clockwise". One of them has to be wrong.